doc/DEVELOPMENT.rst

```diff
@@ -169,7 +169,7 @@ tools:
 
 ::
 
-    brew install meson cmake pkg-config docutils
+    brew install meson cmake pkg-config docutils imagemagick pygments python3
```

Comment:
meson requires python3, so maybe the later is redundant? Unless brew packages it weirdly; I wouldn't know anything about that.